taxonomy/forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+Taxonomy forms
+"""
+
+from django.forms import (
+    ModelForm,
+    TextInput,
+    CheckboxSelectMultiple,
+    RadioSelect,
+)
+
+from .models import Vocabulary
+
+
+class VocabularyForm(ModelForm):
```

Comment:
Can you give an example? I think maybe the [help_text](https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/) could be what you're looking for.